frontend/pages/questions/index.tsx

```diff
@@ -0,0 +1,131 @@
+'use client'
+
+import { Payment, columns } from "@/components/questions/columns"
+import { DataTable } from '@/components/questions/data-table'
+import { useEffect, useState } from 'react';
+
+async function getData(): Promise<Payment[]> {
+    // Fetch data from your API here.
+    return [
+        {
+            id: 1,
+            email: "m@example.com",
+            category: "Algorithms",
+            status: "failed",
+            description: "This is a description",
+            difficulty: "medium"
+          },
+          {
+            id: 1,
+            email: "m@example.com",
+            category: "Algorithms",
+            status: "failed",
```

Comment:
could u make one of the status as success so we can see how to check mark renders